src/support/azext_support/tests/latest/test_support_scenario.py

```diff
@@ -401,3 +402,143 @@ def _validate_failure_rsp(self, rsp, exit_code):
         self.assertTrue(rsp is not None)
         self.assertTrue(rsp.exit_code is not None)
         self.assertTrue(rsp.exit_code == exit_code)
+
+    def test_support_file_attachment_no_subscription(self):
+        # Create File workspace
+        file_workspace_name = self.create_random_name(prefix='cli', length=20)
+        create_file_workspace_result = self.cmd('support no-subscription file-workspace create --file-workspace-name ' + file_workspace_name).get_output_in_json()
+        self.assertTrue(create_file_workspace_result is not None)
```

Comment:
Assert should go in its own validation. Follow the pattern used for other tests

Reply:
Not sure why this was marked as resolved, assert needs to go in its own validation method like we do for other methods